### DISCUSSION TITLE
Error circularly references [@types/geojson]

### DISCUSSION THREAD

**Discussion by rafael-meneses** [DefinitelyTyped/DefinitelyTyped#61246] [Issues with a @types package] [answered]
After the last update of the @types/geojson package, version 7946.0.10, we started receiving this build error.

```
typescript: node_modules/@types/geojson/index.d.ts, line: 79 
            Type alias 'Geometry' circularly references itself. 
      L79:   * MultiPoint geometry object.
      L80:   *  https://tools.ietf.org/html/rfc7946#section-3.1.3
```

[a comment by an automated account is not shown]

**Comment by bondar-artem** [comment]
I have exactly the same issue!

**Reply by icholy** [reply]
What version of TypeScript are you using?

**Reply by bondar-artem** [reply]
3.4.5

**Reply by rafael-meneses** [reply]
2.8.3

**Reply by bondar-artem** [reply]
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/61254 also I have opened a defect

**Reply by icholy** [reply]
Either update your TypeScript version or downgrade the geojson types.

**Reply by bondar-artem** [reply]
which is the next compatible TypeScript version?

**Reply by icholy** [reply]
I think 3.7

**Reply by bondar-artem** [reply]
Unfortunately, my application is on Angular 8, and Angular compiler does not allow me to use anything about 3.6.0
How to downgrade the geojson types? I don't see any dependency related to that in my package.json
Here: https://github.com/Postavshik/ngx-cypress-test/blob/master/package.json

**Reply by icholy** [reply]
Add the older version as an explicit dependency and start using a `package-lock.json` so this type of thing doesn't happen anymore.
